apps/drec-ui/src/api/config.ts

```diff
@@ -0,0 +1,20 @@
+import axios, { AxiosRequestConfig } from 'axios';
+import { LocalStorageKeys } from 'utils/enumExports';
+
+export const useAxiosInterceptors = () => {
+    axios.interceptors.request.use((config: AxiosRequestConfig): AxiosRequestConfig => {
```

Comment:
You can use `defaults` instead of `interceptors`. 
You can see the implementation in `ui-v2`